libbeat/docs/shared-directory-layout.asciidoc

```diff
@@ -24,8 +24,10 @@ The directory layout of an installation is as follows:
 | logs   | The location for the logs created by {beatname_uc}. | {path.home}/logs | path.logs
 |=======================================================================
 
+ifndef::serverless[]
 You can change these settings by using CLI flags or setting
 <<configuration-path,path options>> in the configuration file.
+endif::serverless[]
```

Comment:
@dedemorton At first I was confused about the serverless usage here, maybe because functionbeat previous name was serverless* but It make sense to use a more generic term so we can reuse that flag.

Reply:
Yah I plan to clean up the conditional coding at some point anyhow to improve our naming...we can revisit the question at that point.